testUtils/createEnv.js

```diff
@@ -42,9 +42,34 @@ module.exports = (params) => {
 			}
 		})
 		.then(() => {
-			const migrationsDir = pathUtils.join(dir, 'migrations');
+			let migrationsDirName = 'migrations';
+			let sourceDirName = 'migrations';
 
-			return createEastrc({dir, configParams: {dir: migrationsDir}});
+			const eastrc = {};
+
+			if (params.migratorParams && params.migratorParams.configureParams) {
```

Comment:
if I'm not mistaken all params except `dir` (which is required to put `.eastrc` in right place) could be passed via `params.migratorParams.configureParams` (which will be passed to `migrator.configure`).

Reply:
I removed `migration*Extension` setup from here and left only `dir` and `sourceDir` propagation to test migrator in https://github.com/okv/east/pull/54/commits/bc6b8ad847d5b3bd97ff2234dc259d6669b9a956

Reply:
that's ok